.github/workflows/test.yaml

```diff
@@ -67,3 +68,18 @@ jobs:
           --plugin_path /rbpf/target/release/examples/rbpf_plugin \
           --exclude_regex "${{ env.KNOWN_FAILURES }}"  \
           --plugin_options "--jit" || true
+
+      - name: Install development tools
+        uses: taiki-e/install-action@v1.14.2
+        with:
+          tool: cargo-llvm-cov
+
+      - name: Generate coverage report
+        run: |
+          cargo llvm-cov --lcov --all --all-features --all-targets > lcov.info
+
+      - name: Upload coverage to coveralls
+        uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We don't use the token at the moment, so we can probably remove this line?

Reply:
The github-token is required as per: https://github.com/coverallsapp/github-action

Reply:
Ok I see, sorry for misleading you on that one. Thanks!